schedulingApp/unit-tests/test_accounts.py

```diff
@@ -1,47 +1,45 @@
 from django.test import TestCase
-from models import User
+from django.contrib.auth.models import User
 
 
 # Fields: EMail Phone Address FirstName LastName PermissionLevel OfficeHours
 class CreateAccount(TestCase):
-    def test_emptyUsernameAndPassword(self):
-        a = User()
-        self.assertRaises(TypeError, msg="Should raise TypeError for no arguments, requires a username and password")
-
     def test_emptyUsername(self):
-        a = User("", "", "dogs123")
-        self.assertRaises(TypeError, msg="Should raise TypeError for no arguments, "
-                                         "requires a username with the password")
+        with self.assertRaises(TypeError,
+                               msg="Should raise TypeError for no arguments, requires a first and last name"):
+            a = User("", "")
+
+    def test_emptyFirstName(self):
+        with self.assertRaises(TypeError,
+                               msg="Should raise TypeError for no arguments, requires a first name"):
+            a = User("", "Smith")
 
-    def test_emptyPassword(self):
-        a = User("Robert", "Smith", "")
-        self.assertRaises(TypeError, msg="Should raise TypeError for no arguments, "
-                                         "requires a password for the username")
+
+    def test_emptyLastName(self):
+        with self.assertRaises(TypeError,
+                               msg="Should raise TypeError for no arguments, requires a last name"):
+            a = User("Robert", "")
 
     def test_conflictingUsername(self):
```

Comment:
I don't think this should throw an error; rather, if a user with the same email address is added. There could be multiple people named "John Smith" in the system, for example.